controllers/gitrepository_controller_test.go

```diff
@@ -149,6 +149,14 @@ var (
 	testGitImplementations = []string{sourcev1.GoGitImplementation, sourcev1.LibGit2Implementation}
 )
 
+func transportNotInitialized() bool {
+	return false
+}
+
+func transportInitialized() bool {
+	return true
+}
```

Comment:
Let's add `mock` prefix on this functions to make it clear that they are just mocking.